### ISSUE TITLE
update fabric for OFU/GFU (Trac #471)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#470]
16:23 <schlemihl> i did
16:23 <schlemihl> pnf@fpslave01[~]% mkdir -p 
                  /mnt/data/pnflocal/opticalfollowup/alerts
16:23 <schlemihl> pnf@fpslave01[~]% mkdir -p 
                  /mnt/data/pnflocal/opticalfollowup/spade
16:23 <schlemihl> pnf@fpslave01[~]% mkdir -p /usr/local/pnf/gammafollowup/events
16:23 <schlemihl> pnf@fpslave01[~]% mkdir -p 
                  /usr/local/pnf/gammafollowup/events_blind
16:23 <schlemihl> plus
16:23 <schlemihl> pnf@fpslave01[~]% ln -s /mnt/data/pnflocal/opticalfollowup 
                  opticalfollowup

Symlink might not be needed

Migrated from https://code.icecube.wisc.edu/ticket/471
```json
{
    "status": "closed", 
    "changetime": "2019-01-11T21:46:03", 
    "description": "16:23 <schlemihl> i did\n16:23 <schlemihl> pnf@fpslave01[~]% mkdir -p \n                  /mnt/data/pnflocal/opticalfollowup/alerts\n16:23 <schlemihl> pnf@fpslave01[~]% mkdir -p \n                  /mnt/data/pnflocal/opticalfollowup/spade\n16:23 <schlemihl> pnf@fpslave01[~]% mkdir -p /usr/local/pnf/gammafollowup/events\n16:23 <schlemihl> pnf@fpslave01[~]% mkdir -p \n                  /usr/local/pnf/gammafollowup/events_blind\n16:23 <schlemihl> plus\n16:23 <schlemihl> pnf@fpslave01[~]% ln -s /mnt/data/pnflocal/opticalfollowup \n                  opticalfollowup\n\n\nSymlink might not be needed", 
    "reporter": "blaufuss", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547243163444312", 
    "component": "jeb + pnf", 
    "summary": "update fabric for OFU/GFU", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2013-09-16T20:26:16", 
    "milestone": "", 
    "owner": "blaufuss", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150211T19:40:53: 
* blaufuss changed status from "new" to "closed"
* blaufuss commented:

> Will drop off pole, with new siglet stream..


* blaufuss changed resolution from "" to "wontfix"

**Comment by icecube-trac** [comment]
Trac update at 20150722T18:43:31: 
* david.schultz changed resolution from "wontfix" to "fixed"
* david.schultz changed _comment0 from:

> In [changeset:"2176/IceTray"]:
> {{{
> #!CommitTicketReference repository="IceTray" revision="2176"
> add a 'go to event' feature. fixes #471
> }}}

to:

> 1437590738607030

**Comment by icecube-trac** [comment]
Trac update at 20190111T21:46:03: nwhitehorn commented:

> In changeset 169146:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="169146"
> add a 'go to event' feature. fixes #470
> ```
